StripeIdentity/StripeIdentity/Source/Helpers/String+Localized.swift

```diff
@@ -20,11 +20,18 @@ extension String.Localized {
 
     static var date_of_birth: String {
         STPLocalizedString(
-            "Date of birth",
+            "Date of Birth",
             "Label for Date of birth field"
         )
     }
 
+    static var date_of_birth_invalid: String {
+        STPLocalizedString(
+            "Date of birth does not look valid",
+            "Message for invalid Date of birth field"
```

Comment:
Should those also be capitalized? `Date of Birth`?

Reply:
this is in line with android text, only the section title is capitalized